extension/lib/getAuthenticatedCheckoutUrl.js

```diff
@@ -0,0 +1,26 @@
+const { URL } = require('url')
+const AuthRepository = require('./bigcommerce/AuthRepository')
+
+/**
+ * @param {PipelineContext} context
+ * @param {GetAuthCheckoutInput} input
+ * @returns {Promise<ShopgateGetCheckoutUrlResponse>}
+ */
+module.exports = async (context, input) => {
+  if (!context.meta.userId) {
```

Comment:
Adding jsdoc for context.meta in PipelineContext definition would be nice.

Reply:
this is added